components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -120,6 +121,7 @@ internal fun resetGlean(
     // Init glean.
     Glean.initialized = false
     Glean.initialize(context, config)
+    CountersStorageEngine.clearAllStores()
```

Comment:
Why is this needed? We're clearing all the stores at the beginning of this function already. If there's a good reason to do that, would you kindly add a line of comment about why (so that we don't forget about it?)?

Reply:
Oops.  It's not needed.